acceptance/bin/extract_command_exec_id.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+import json
+import re
+import sys
+from pathlib import Path
+
+
+def extract_cmd_exec_id():
```

Comment:
This function is called extract_cmd_exec_id() but it returns an error and prints cmd_exec_id(), which is somewhat unexpected.

You can raise the errors with 'raise SystemExit( "Requests file is empty")' instead and either return extracted cmd_exec_id or print it to stdout.